.gitignore

```diff
@@ -78,3 +78,4 @@ tmp*
 /clients/python/artifacts/*
 /clients/python/client/build/*
 /clients/python/client/osparc/data/openapi.json
+/clients/python/test/e2e/pyproject.toml
```

Comment:
Yes, I realize this is a bit unusual. But the point is that this toml will have to contain the (key, secret) pair for accessing the server. A lot of the "trouble" (or maybe better "challenge") in this pr is to deal with the fact that I cannot just write my toml file and commit it into the repo. It must be generated on the ci. 
The basic idea I had was to use the `pytest-env` plugin to define environment variables directly in the toml file rather than simply exposing them in the environment "by hand". This is nice for reproducibility. The toml file is uploaded as an artifact on the ci, and from that file one can reproduce the whole thing. But it also means it should never be committed into the repo.